hummingbot/client/ui/hummingbot_cli.py

```diff
@@ -42,12 +48,16 @@ def _handle_exception_patch(self, loop, context):
 Application._handle_exception = _handle_exception_patch
 
 
-class HummingbotCLI:
+class HummingbotCLI(PubSub):
+    class Event(Enum):
+        START = 1
```

Comment:
I think we should include this new event into `core.event.events.py` instead of having it in `hummingbot_cli.py`